.github/actions/copy-workflow-go/action.yml

```diff
@@ -46,6 +46,27 @@ runs:
             # We don't tidy, because the next step does that.
             # Separate commits also help with reviews.
             git commit -m "bump go.mod to Go $TARGET_VERSION and run go fix"
+
+            # As of Go 1.19 io/ioutil is deprecated
+            # We automate its upgrade here because it is quite a widely used package
+            if [[ $TARGET_VERSION == "1.18" ]]; then
```

Comment:
This `if` is always true, isn't it?

Reply:
That's correct. I wanted to make it clear that we can remove it in time for the next upgrade but I guess a comment might be more fitting for that.

Reply:
We probably shouldn't remove it. If any of the updates is not merged (and the repo stays on an old uCI version), it would be nice if this was included in the next update, just in case. wdyt?

Reply:
You're right, we might also want to enrol a new repository that's not on latest Go yet and still uses `ioutil` - it'd come in handy then too.

Let me just remove the if-clause then.